### ISSUE TITLE
Which file do I have to edit to get the grid layout

### ISSUE THREAD

**Issue by lodusingh** [TelescopeJS/Telescope#1484]
Hey, I'm new to the telescope and trying to understand its architecture but couldn't figure which file contains the code that is responsible for the front-end of posts.

I know that it's somewhere in the directory

project/packages/nova-base-components/lib/posts

but I became overwhelmed by the number of files and couldn't really figure out where and how to turn posts from list view to grid view.
Thanks :)

**Comment by xavcz** [comment]
Hey! You should have a look at this video series explaining how to customize components and style of Telescope Nova: https://www.youtube.com/playlist?list=PLBoa_Q6hVeSzChsSi8X8x3FQ8NN2D73fB

To have a better understanding of the component tree and so what file is what, I highly recommend using React DevTools in your browser, see screenshot here: https://slack-files.com/T02L990A7-F224SV0D6-ce86fdf720

Basically, to do what you want to do, you'll have a custom package with a copy of base-styles that you modify directly and some extension of existing components (see the videos above and the Readme).

Feel free to drop by the Slack room if you need help 👌

**Comment by SachaG** [comment]
Did you check out the videos on http://www.telescopeapp.org/ ? Apart from
that the best technique is to use the React devtools to inspect the Virtual
DOM and find out the name of each component.
ᐧ
